<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maindosen</class>
 <widget class="QMainWindow" name="Maindosen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>160</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnPageAcc">
       <property name="text">
        <string>AccMahasiswa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPageCari">
       <property name="text">
        <string>Cari Mahasiswa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPageTampil">
       <property name="text">
        <string>Tampilkan Mahasiswa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnKembali">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>381</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="pagetambah">
     <widget class="QPushButton" name="btnAcc">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tampilqueue">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>131</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageCari">
     <widget class="QLineEdit" name="inputnim">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCari">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Cari</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tampilmahasiswa">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>151</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NIM</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="btnTampilMhs">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Tampilkan</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tampilmhs">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>301</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
